docs/user-guide/commands.md

```diff
@@ -341,6 +366,7 @@ Flag | Help
 `-t, --to DATE` | The date at which the report should stop (inclusive). Defaults to tomorrow.
 `-p, --project TEXT` | Reports activity only for the given project. You can add other projects by using this option several times.
 `-T, --tag TEXT` | Reports activity only for frames containing the given tag. You can add several tags by using this option multiple times
+`-j, --json` | Outputs the report as JSON instead of writing to the console
```

Comment:
JSON output still goes to the console, doesn't it? Just the format changes from formatted text with escape sequences to JSON. How can this be reworded?